lesson-3/script.js

```diff
@@ -0,0 +1,109 @@
+let dateUser;
+let dateUserIsNumber;
+let userAge;
+
+function usersBirthYear() {
+
+  function validateUserBirthDate() {
+    return (
+      !dateUser || dateUser < 1900 || dateUser > 2023 || isNaN(dateUserIsNumber)
+    );
+  }
+  
+  do {
+    dateUser = prompt("Enter please your birth date");
+    dateUserIsNumber = Number(dateUser);
+    
+    if (validateUserBirthDate(dateUser)) {
+      alert("It is a pity that you did not want to enter your birth date...");
+    }
+  } while (validateUserBirthDate(dateUser));
+  
+
+  userAge = new Date().getFullYear() - dateUser;
+};
+
+function validateUserString (validate) {
+  if (!validate || validate == ' ' || validate.trim() == '') {
```

Comment:
Don't use non strict equality / inequality (only strict `===` / `!==`)
```suggestion
  if (!validate || validate.trim() === '') {
```